lighthouse-core/audits/image-aspect-ratio.js

```diff
@@ -40,7 +41,10 @@ class ImageAspectRatio extends Audit {
     const url = URL.elideDataURI(image.src);
     const actualAspectRatio = image.naturalWidth / image.naturalHeight;
     const displayedAspectRatio = image.width / image.height;
-    const doRatiosMatch = Math.abs(actualAspectRatio - displayedAspectRatio) < THRESHOLD;
+
+    const targetDisplayHeight = image.width / actualAspectRatio;
+    const doRatiosMatch = Math.abs(actualAspectRatio - displayedAspectRatio) < THRESHOLD_PERCENT ||
```

Comment:
is there value in maintaining the ratio difference check, then? It seems like it would either be less lenient than the pixel check or the images would be big enough that they would have the flexibility to change the image size to get under the limit.

I haven't completely worked out the problem, though, so I may be missing cases :)

Reply:
a quick check with this branch shows that dropping it for the pixel check would still pass all our tests, at least :)

Reply:
yeah sgtm, let's just go with targetDisplayHeight